pkg/services/ngalert/api/tooling/post.json

```diff
@@ -2882,7 +2886,7 @@
      "type": "boolean"
     },
     "settings": {
-     "$ref": "#/definitions/Json"
+     "$ref": "#/definitions/RawMessage"
```

Comment:
It looks like we forgot to update the specs in https://github.com/grafana/grafana/pull/71692

Reply:
True, looks like the this is paying dividends already!